esmvaltool/preprocessor/_derive/__init__.py

```diff
@@ -0,0 +1,118 @@
+"""Automatically derive variables."""
+
+
+import importlib
+import logging
+import os
+
+import iris
+import yaml
+
+from ._derived_variable_base import DerivedVariableBase
+
+logger = logging.getLogger(__name__)
+
+ALL_DERIVED_VARIABLES = {}
+
+
+def get_required(short_name, field=None):
+    """Get variable short_name and field pairs required to derive variable.
+
+    It is also possible to process fx variables using the tuple ('fx_files',
```

Comment:
This is confusing, why not simply add third entry to the returned tuple, i.e. return (short_name, field, list_of_fx_files)
or if you think that's not clear, return a dict with entries short_name, field, fx_files?